tests/test_cellbase_records.py

```diff
@@ -8,6 +8,7 @@ class CellbaseRecordsTest(unittest.TestCase):
     def setUpClass(cls):
         cls.cb_records = cellbase_records.CellbaseRecords()
 
+    @unittest.skip("SKIPPING test cellbase records")
```

Comment:
How long does this actually take? It could be enough to test just one request, and checks a record has the fields the application requires.